doc/source/conf.py

```diff
@@ -478,6 +479,8 @@
 ]
 
 
+remove_from_toctrees = ["reference/*"]
```

Comment:
@jorisvandenbossche could you recommend which files I should be removing toctrees from?